### ISSUE TITLE
Compilation error: missing 

### ISSUE THREAD

**Issue by dov** [stripe2933/vk-gltf-viewer#7]
I followed the instructions in the README.md but when compiling it failed with the following error:

```
/terra/space/pub-repos/vk-gltf-viewer/build/vcpkg_installed/x64-linux-clang/module/vku/extlibs/module-ports/vk_mem_alloc.cppm:3:10: fatal error: 'vulkan-memory-allocator-hpp/vk_mem_alloc.hpp' file not found
```

It seems like the vku vcpkg is missing the dependency on vulkan-memory-allocator-hpp. Is there a work around?

**Comment by stripe2933** [comment]
> It seems like the vku vcpkg is missing the dependency on vulkan-memory-allocator-hpp.

https://github.com/stripe2933/vk-gltf-viewer/blob/b236be8eac78bd1d1736624d7d6674619c964c63/overlays/vku/vcpkg.json#L17

My guess is that your CMake project is using the `include` directory of VulkanMemoryAllocator-Hpp installed on your system, rather than the one provided by vcpkg. Could you check if the `build` folder contains the file `vcpkg_installed/<your vcpkg triplet name>/include/vulkan-memory-allocator-hpp/vk_mem_alloc.hpp`?

**Comment by dov** [comment]
Thanks for your reply. I checked, and the build folder does contain `vcpkg_installed/x64-linux-clang/include/vulkan-memory-allocator-hpp/vk_mem_alloc.hpp`. I also checked that there is no other plausible version of `vk_mem_alloc.hpp` that can have been mistakenly included. Strange...

**Comment by stripe2933** [comment]
Yeah, that’s strange. Could you upload the full vcpkg configuration log file?

**Comment by dov** [comment]
Attached below, thanks! 
[vcpkg-manifest-install.log](https://github.com/user-attachments/files/17270543/vcpkg-manifest-install.log)
